src/main/java/entity/player/CommonPlayer.java

```diff
@@ -1,16 +1,23 @@
 package entity.player;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import entity.Card;
+
 /**
- * A simple implementation of the User interface.
+ * A simple implementation of the Player interface.
  */
 public class CommonPlayer implements Player {
 
     private final String playerID;
     private final String password;
```

Comment:
@EstelleWang-UofT we made the decision to get rid of all lab5 code, incl. anything regarding signup, login, etc. So this password field and related methods can be removed.